Resources/config/security.xml

```diff
@@ -6,7 +6,6 @@
 
     <parameters>
         <parameter key="fos_user.security.interactive_login_listener.class">FOS\UserBundle\EventListener\LastLoginListener</parameter>
-        <parameter key="fos_user.security.login_manager.class">FOS\UserBundle\Security\LoginManager</parameter>
```

Comment:
@Soullivaneuh please don't remove existing parameters. This is a BC break, so I would do it all at once, not service by service